ansible/main.yml

```diff
@@ -48,6 +48,20 @@
   roles:
     - { role: okfn-org-manager, tags: ['heroku-backup'] }
 
+- hosts: s138.okserver.org
+  sudo: true 
```

Comment:
The intention is to have infra and credentails on the same folder level. We already do this `bootstrap.yml`. I'm strongly in favor of things just working. Doing this also means, we can kill off the `play` script.